packages/fxa-payments-server/src/utils.js

```diff
@@ -0,0 +1,74 @@
+import { ActionType as PromiseActionType } from 'redux-promise-middleware';
+const { assign } = Object;
+
+export const parseParams = params => params
```

Comment:
This could use [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) to parse.

Reply:
Hmm, I've used that on other projects but thought it was relatively new. Looks like it's got pretty decent browser support these days